code/modules/surgery/organs/robolimbs.dm

```diff
@@ -3,142 +3,219 @@ GLOBAL_LIST_EMPTY(chargen_robolimbs)
 GLOBAL_LIST_EMPTY(selectable_robolimbs)
 GLOBAL_DATUM(basic_robolimb, /datum/robolimb)
 
+
+#define model        0
+#define brand        1
+#define childless    2
```

Comment:
These could likely use some comments or renaming.

- Model = This iconset contains _only_ a monitor and is a `subtypeof` a `brand`.
- Brand = This iconset contains all body parts including a monitor _and_ there are other monitor `model`s that fall under this `brand`.
- Childless = This iconset contains all body parts including a monitor. There are no other monitors for this `brand`.


The reason Brand exists is to reduce menu options for the IPC head selection. It's so you can go:
  Limb -> Head -> Prothesis -> Brand -> Handful of Monitor Models
Rather than:
  Limb -> Head -> Prothesis -> Dozens of Monitor Models
  
I did experiment with a "parent" system (e.g. but that ended up creating additional menu bloat.